rpc/common/service.proto

```diff
@@ -54,6 +54,7 @@ message Package {
   string            digest      = 16;
   bool              dev         = 17;
   bool              indirect    = 18;
+  string            maintainer = 21;
```

Comment:
nit: Did you run `mage fmt`?

Reply:
No, I just did it and updated the pr, can you please check it once ? Thank you!!